atoma-state/src/migrations/20250411093352_add-tx-digest-to-stacks.sql

```diff
@@ -0,0 +1,8 @@
+-- Add tx_digest column to stacks table
+ALTER TABLE
```

Comment:
This formatting looks a bit weird to me, can we follow the usual formatting ?

Reply:
24c7dcf